packages/loot-core/src/mocks/index.js

```diff
@@ -5,9 +5,9 @@ export function generateAccount(name, isConnected, type, offbudget) {
   return {
     id: uuid.v4Sync(),
     name,
-    balance_current: isConnected ? (Math.random() * 100000) | 0 : null,
-    bank: isConnected ? (Math.random() * 10000) | 0 : null,
-    bankId: isConnected ? (Math.random() * 10000) | 0 : null,
+    balance_current: isConnected ? Math.floor(Math.random() * 100000): null,
+    bank: isConnected ? Math.floor(Math.random() * 10000): null,
+    bankId: isConnected ? Math.floor(Math.random() * 10000): null,
```

Comment:
Just a tiny suggestion:

```suggestion
    balance_current: isConnected ? Math.floor(Math.random() * 100000) : null,
    bank: isConnected ? Math.floor(Math.random() * 10000) : null,
    bankId: isConnected ? Math.floor(Math.random() * 10000) : null,
```

Reply:
Thanks! Hopefully in future we can get these sort of things handled automatically with https://github.com/actualbudget/actual/pull/130